governator-test/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -94,6 +96,20 @@ public InjectorCreationMode getInjectorCreationMode() {
     protected LifecycleInjector createInjector(List<Module> modules, List<Module> overrides) {
         return InjectorBuilder.fromModules(modules).overrideWith(overrides).createInjector();
     }
+    
+    protected MockHandler createMockHandlerForTestClass(Class<?> testClass, Class<? extends MockHandler> defaultMockHandler) {
```

Comment:
Do we want to allow scanning of base classes as well or is using inheritance for tests an anti-pattern?